components/Materiallist.js

```diff
@@ -0,0 +1,109 @@
+import styled from "styled-components";
+import SwipeToDeleteItem from "./SwipeToDelete";
+import MaterialListForm from "./MaterialListForm";
+
+export default function MaterialList({
+  addNewMaterial,
+  currentEntry,
+  handleMateriallistDelete,
+}) {
+  return (
+    <>
+      <MaterialListForm addNewMaterial={addNewMaterial} />
+      <FlexWrapper>
+        <ListHeader>p</ListHeader>
+        <ListHeader>w</ListHeader>
+        <ListHeader>h</ListHeader>
+        <ListHeader>d</ListHeader>
+        <ListHeader>unit</ListHeader>
+      </FlexWrapper>
+      <List>
+        {currentEntry.dimensions.map((dimension) => (
+          <li key={dimension.id}>
+            <SwipeToDeleteItem
+              handleMateriallistDelete={() =>
+                handleMateriallistDelete(dimension.id)
+              }
+              dimensionId={dimension.id}
+            >
+              <StyledSubList>
+                <Wrapper>
+                  <ListItem>
```

Comment:
Your wrapper is not a list element. Only use li elements inside of list elements